### PR TITLE
Fixes #15809: Cannot update editor techniques at webapp startup if the request to get techniques timeout (5 seconds)

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/15809

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
Commit modified

File: webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/ncf/TechniqueWriter.scala [hunk 1]
```diff
@@ -193,15 +193,19 @@ class TechniqueWriter (
   }
 
   // Write and commit all techniques files
-  def writeAll(technique : Technique, methods: Map[BundleName, GenericMethod], modId : ModificationId, committer : EventActor) : Result[Seq[String]] = {
+  def writeAll(technique : Technique, methods: Map[BundleName, GenericMethod], modId : ModificationId, committer : EventActor, launchLibUpdate : Boolean) : Result[Seq[String]] = {
```
Comment (fanf) [inline webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/ncf/TechniqueWriter.scala]: boolean option are hard to understand (we should never have one I believe). I think we should have two methods here: `writeAll` and `writeAllAndUpdateTechLib` and just call the correct one in either place.

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by fanf** [comment]
OK, merging this PR

**Pull request merged**